.github/workflows/push-trigger.yml

```diff
@@ -0,0 +1,81 @@
+name: Maven Package upon a push
+
+on:
+  release:
+    types: [published]
+  pull_request:
+    types: [opened]
+  workflow_dispatch:
+    inputs:
+      message:
+        description: 'Message for manually triggering'
+        required: false
+        default: 'Triggered for Updates'
+        type: string
+  push:
+    branches:
+      - '!release-branch'
+      - '*'
```

Comment:
Please update the branches as discussed in standup.
MOSIP*
1.*
!release-branch
develop
master